### DISCUSSION TITLE
Local dev with multiple teams users

### DISCUSSION THREAD

**Discussion by irwinwilliams** [microsoft/hack-together-teams#61] [Q&A]
For my team's scenarios, we need to test notifying specific users (rather than a broadcast).
I've been finding that local dev configuration is only allowing one user on my machine to install the app, when I try using the url with a fresh user (both are Global Admins), no dice.
I've tried Debugging to Chrome - I've been defaulting to Edge - but that browser opens and closes immediately. 
Would welcome any guidance on this.

**Comment by aycabas** [comment]
Hi @irwinwilliams, Teams Toolkit allows you to preview your app on your local machine. Once you are done with debugging and building your app you'll need to publish the app on Teams Store (for public usage) or upload your app on your tenant (for internal usage), so everyone in your organization can use the app. 

I hope I understand your question correctly, let me know if you have any more questions that I can help with.
